styles.css

```diff
@@ -134,3 +134,694 @@ h3 {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  margin-left: 24px;
+}
+
+.service-button {
+  list-style-type: none;
+  margin-top: 12px;
+}
+
+.css-btn {
+  width: 49px;
+  height: 40px;
+  padding: 10px, 12px, 10px, 12px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+  text-decoration: none;
+  justify-content: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.html-btn {
+  width: 56px;
+  height: 40px;
+  padding: 10px, 12px, 10px, 12px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+  margin-left: 10px;
+  text-decoration: none;
+  justify-content: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.bootstrap-btn {
+  width: 95px;
+  height: 40px;
+  padding: 10px, 12px, 10px, 12px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+  margin-left: 10px;
+  text-decoration: none;
+  justify-content: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.ruby-btn {
+  width: 61px;
+  height: 40px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  text-decoration: none;
+  justify-content: center;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+  margin-left: 10px;
+}
+
+#css-btn {
+  width: 25px;
+  height: 20px;
+  padding: 10px, 12px;
+  color: #091e42;
+  align-items: center;
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  margin-top: 8px;
+}
+
+#html-btn {
+  width: 32px;
+  height: 20px;
+  top: 10px;
+  margin-left: 1px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  list-style-type: none;
+  text-decoration: none;
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 8px;
+  padding: 10px, 12px, 10px, 12px;
+}
+
+#bootstrap-btn {
+  width: 71px;
+  height: 20px;
+  top: 10px;
+  margin-left: 1px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  text-decoration: none;
+  margin-top: 8px;
+}
+
+#ruby-btn {
+  width: 37px;
+  height: 20px;
+  top: 10px;
+  margin-left: 1px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  text-decoration: none;
+  margin-top: 8px;
+}
+
+.See-project-btn {
+  width: 124px;
+  height: 48px;
+  margin-top: 18px;
+  background-color: #ff6b00;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  text-align: center;
+  padding: 0 1px;
+  margin-left: 24px;
+  font-size: 17px;
+  line-height: 3px;
+  outline: none;
+  border: none;
+}
+
+.printing {
+  background-image: url(./images/probg.png);
+  background-position: 100%;
+  background-repeat: no-repeat;
+  width: 327px;
+  margin-top: 42px;
+  height: 386px;
+  top: 8px;
+  margin-left: 16px;
+  background-color: linear-gradient rgba(38, 38, 38, 0) 0.34% #262626 61.21%;
+}
```

Comment:
- [ ] Kindly adjust the project cards to centralize them on the page as given in the design.
- your design 👇 
![tl](https://user-images.githubusercontent.com/60146030/219865642-b61d2937-f053-495b-8e32-e80fb26fe087.PNG)
- Figma design 👇 
![fg](https://user-images.githubusercontent.com/60146030/219865666-daabf3c2-b6ba-4b73-a0d9-460e64143865.PNG)

Reply:
Done, please check now